pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -61,6 +61,9 @@ var istioMtlsTransportSocketMatch = &structpb.Struct{
 	},
 }
 
+// SupportedEcdhCurves for upstream TLS configuration.
```

Comment:
I think this may break users compiling with FIPS mode in envoy - it will likely reject these. I can test it.

Have we looked into getting envoy to increase the default? That will make our config size smaller, trivially add the check for ^. And if they ah no maybe they give us a good reason that we shouldn't?